src/database/migrations/20190901120620-adding_to_user.js

```diff
@@ -0,0 +1,51 @@
+export default {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.addColumn('Users', 'gender', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'birthday', {
+      type: Sequelize.DATE,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'preferredlanguage', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'currency', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'residentialaddress', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'role', {
+      type: Sequelize.STRING,
+      values: [
+        'Developer',
+        'QA',
+        'Maintenance',
+      ],
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'department', {
+      type: Sequelize.STRING,
+      allowNull: true
+    });
+    await queryInterface.addColumn('Users', 'linemanager', {
+      type: Sequelize.INTEGER,
```

Comment:
your line manager column should reference a userId on your table as per @Stylll advice